crypto/thread_test.cc

```diff
@@ -152,6 +152,34 @@ TEST(ThreadTest, RandState) {
   thread.join();
 }
 
+static void thread_task(bool *myFlag) {
+  EXPECT_EQ(1, AWSLC_thread_local_clear());
+  EXPECT_EQ(1, AWSLC_thread_local_clear());
+  uint8_t buf[8];
+  EXPECT_EQ(1, RAND_bytes(buf, sizeof(buf)));
+  EXPECT_EQ(1, AWSLC_thread_local_clear());
+  ERR_clear_error();
+  EXPECT_EQ(1, AWSLC_thread_local_clear());
+  EXPECT_EQ(1, AWSLC_thread_local_clear());
```

Comment:
why call `AWSLC_thread_local_clear` twice here?

Reply:
The concern is that `AWSLC_thread_local_clear` could leave the thread-local data in an invalid state, causing an (e.g.) seg-fault if the function is called a second time.